Nos/Controller/SearchController.swift

```diff
@@ -50,12 +60,28 @@ class SearchController: ObservableObject {
                 }
                 return query
             }
-            .map { self.authors(named: $0) }
+            .map { self.authors(named: $0) } // this and below need to run every time the context changes
```

Comment:
@mplorentz I don't remember what to do about this comment, if anything. I remember we discussed having separate chains but don't understand the code well enough to know how to do that.